pyiron/sphinx/base.py

```diff
@@ -614,22 +616,57 @@ def _set_kpoints(
             raise ValueError("manual_kpoints is not implemented in SPHInX yet")
         if weights is not None:
             raise ValueError("manual weights are not implmented in SPHInX yet")
-        if scheme != "MP":
-            raise ValueError("only Monkhorst-Pack mesh is implemented in SPHInX")
-        if mesh is not None:
-            self.input["KpointFolding"] = (
-                "[" + str(mesh[0]) + ", " + str(mesh[1]) + ", " + str(mesh[2]) + "]"
-            )
-        if center_shift is not None:
-            self.input["KpointCoords"] = (
-                "["
-                + str(center_shift[0])
-                + ", "
-                + str(center_shift[1])
-                + ", "
-                + str(center_shift[2])
-                + "]"
+        if scheme == "MP":
+            self._kpoints_odict = None
+            if mesh is not None:
+                self.input["KpointFolding"] = (
+                        "[" + str(mesh[0]) + ", " + str(mesh[1]) + ", " + str(mesh[2]) + "]"
```

Comment:
How about writing something like `np.array2string(mesh, separator=',')`?

Reply:
Sure, but I didn't implement that function.

Reply:
done